vernac/g_vernac.mlg

```diff
@@ -581,6 +581,12 @@ GRAMMAR EXTEND Gram
       | IDENT "Collection"; id = identref; ":="; expr = section_subset_expr ->
           { VernacNameSectionHypSet (id, expr) }
 
+      (* Requiring an external file *)
+      | IDENT "From" ; ns = global ;
+        IDENT "Extra"; IDENT "Dependency"; f = ne_string ;
+        id = OPT [ "as"; id = IDENT -> { id } ] ->
+          { VernacExtraDependency (ns, f, Option.map Id.of_string id) }
```

Comment:
Should the command accept multiple dependencies as "From ... Require" does?  Or is the number of dependencies likely to be small?

Reply:
I think one could iterate `"foo" as x, "bar" as y, ...". It is not a bad idea, but I don't have a lot of energy now, so since the proposed extension is backward compatible I'd leave it to another PR.